recipes/allennlp/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set name = "allennlp" %}
+{% set version = "0.4.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dbc0b7de268a14ad978ad29ceb785c0a22cab13e2786f04fe933b4c71e9ad7aa" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  # Fix this when the new release is officially out.
+  url: https://github.com/allenai/allennlp/archive/v0.4.3.tar.gz
```

Comment:
Would template version in this URL to avoid accidentally publishing something later where the URL and version point to different things.